### PR TITLE
Adds a command-line argument to support choosing a specific merged PR

### PR DESCRIPTION
**Pull request by orta**
I'm on the side of  we shouldn't support arbitrary repos e.g. `danger local --pr=artsy/eigen#11` as the file system isn't going to match the expectations of danger, a merged PR at least will have an approximation of the current file system ( though, obviously it may have changed since then, but that should be pretty obvious to the user. ) 

So this just adds a way to pass a single id along via `danger local --use-merged-pr=11`

Fixes https://github.com/danger/danger/issues/134

### CODE REVIEW

**Comment by jeroenvisser101** [comment]
This needs documentation, but looks good!

**Comment by orta** [comment]
Agreed, updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## Master
+## 0.7.4
```
Comment (jeroenvisser101) [inline CHANGELOG.md]: @orta we should add back # Master?

Comment (orta) [inline CHANGELOG.md]: already have done in the prepare for 0.7.4 stage

Comment (jeroenvisser101) [inline CHANGELOG.md]: Great stuff! :)

**Pull request merged**